deno.json

```diff
@@ -33,12 +32,22 @@
             "*cov_profile*",
             "*tauri*",
             "app/UI/assets/",
-            "*/tailwind.js"
+            "*/tailwind.js",
+            "build-pwa/"
```

Comment:
Why do we need to format the `build-pwa` folder even though it's listed in `gitignore`?

Reply:
We don't need, that's why it's excluded. `.gitignore` and `deno fmt` are separate

Reply:
Okay, I made a mistake.